x-pack/plugins/observability_solution/observability/public/components/alert_search_bar/constants.ts

```diff
@@ -30,6 +31,16 @@ export const ALL_ALERTS: AlertStatusFilter = {
 export const ACTIVE_ALERTS: AlertStatusFilter = {
   status: ALERT_STATUS_ACTIVE,
   query: `${ALERT_STATUS}: "${ALERT_STATUS_ACTIVE}"`,
+  filter: [
+    {
+      query: {
+        match_phrase: {
+          [ALERT_STATUS]: ALERT_STATUS_ACTIVE,
+        },
+      },
+      meta: {},
+    },
+  ],
```

Comment:
it's already specified in query so why needed as separate filter?

Reply:
Alert grouping component accepts filters, not queries, but we use queries in other places. Maybe, in the future, we can refactor places that we use query and use filter instead.